src/App.jsx

```diff
@@ -3,26 +3,96 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos, getUser } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    title: '',
+    status: 'All',
```

Comment:
move this values to the TodoList components state, entire app shouldn't know about filtering values